static/defactor-pools.json

```diff
@@ -1,142 +1,158 @@
+        "collateral": "0xd33c7b753ecaa85e5d5f5b5fd99dec59f26a087e"
+      }
+    ]
+  },
+  {
+    "id": "pools-aurus2-testnet",
+    "name": "Aurus2 Development",
+    "description": "Testnet environment for Aurus2.",
+    "web": "https://aurus2.defactor.dev",
+    "api": "https://aurus2-api.defactor.dev",
+    "testnet": true,
+    "chains": [
+      {
+        "id": "Amoy",
+        "contract": "0x666666ea97809364D68E77c3C3AC036D47d86C93",
+        "token": "0x60E87395b0101F85C623c38Be010574f958496db",
+        "collateral": "0x7F400A79139A603E5F8A930cEC9562171fDCD906"
+      }
+    ]
+  },
+  {
+    "id": "pools-aurus2-mainnet",
+    "name": "Aurus2 Production",
+    "description": "Production environment for Aurus2.",
+    "web": "https://aurus2.factr.app",
+    "api": "https://aurus2-api.factr.app",
+    "testnet": false,
+    "chains": [
+      {
+        "id": "Polygon",
+        "contract": "0x5B23290a4ff67d4A5544935ca89943f00948153A",
+        "token": "0x3c499c542cef5e3811e1192ce70d8cc03d5c3359",
+        "collateral": "0xa6da8c8999c094432c77e7d318951d34019af24b"
+      }
+    ]
+  },
+  {
+    "id": "pools-qba-testnet",
+    "name": "QBA Credit Development",
+    "description": "Testnet environment for QBA Credit.",
+    "web": "",
+    "api": "https://qbacredit-api.defactor.dev",
+    "testnet": true,
+    "chains": [
+      {
+        "id": "Amoy",
+        "contract": "0x0a2e94dd0D8EFa9598F3fa7287F888dc03aDd288",
+        "token": "0x6BaB3bEA9aD00893101837d46638B470076f8AAF",
+        "collateral": ""
+      }
+    ]
+  },
+  {
+    "id": "pools-landx-testnet",
+    "name": "LandX Development",
+    "description": "Testnet environment for LandX.",
+    "web": "https://landx.defactor.dev",
+    "api": "https://landx-api.defactor.dev",
+    "testnet": true,
+    "chains": [
+      {
+        "id": "Sepolia",
+        "contract": "0xC13cB695fCBe2436C4D76aE6CBD2C50a34078E76",
+        "token": "0x75fe3476d90598080f7D12365020C438943Dcef3",
+        "collateral": "0x7F400A79139A603E5F8A930cEC9562171fDCD906"
+      }
+    ]
+  },
+  {
+    "id": "pools-landx-mainnet",
+    "name": "LandX Production",
+    "description": "Production environment for LandX.",
+    "web": "https://landx.defactor.dev",
+    "api": "https://landx-api.defactor.dev",
+    "testnet": false,
+    "chains": [
+      {
+        "id": "Ethereum",
+        "contract": "0x326a7E3cB01f2d429B871114A1417dd3B3cAbfD6",
+        "token": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+        "collateral": "0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48"
+      }
+    ]
+  },
+  {
+    "id": "pools-landshare-testnet",
+    "name": "LandShare Development",
+    "description": "Testnet environment for LandShare.",
+    "web": "https://landshare.defactor.dev",
+    "api": "https://landshare-api.defactor.dev",
+    "testnet": true,
+    "chains": [
+      {
+        "id": "Base",
+        "contract": "0x7D5c1468D8bE9f0F4FaD26F2Cb7e6b2ed9042577",
```

Comment:
```text
FACTR deployed on: 0x7D5c1468D8bE9f0F4FaD26F2Cb7e6b2ed9042577
```

https://sepolia.etherscan.io/address/0x7D5c1468D8bE9f0F4FaD26F2Cb7e6b2ed9042577

That address belongs to `FACTR`, it's wrong.